### PR TITLE
add header to certain pages

### PR DESCRIPTION
**Pull request by solring**
Tiny changes.

I removed the "hi~" button under guest mode, because it's still clickable with no effect, which is kinda confusing?
Or should I change it into "Login" button?

### CODE REVIEW

**Comment by chhsiao1981** [comment]
hi @solring ～

"hi～" in header is supposed to be "hi～[username]"
and clickable to /login if username === PTT_GUEST
otherwise to /user/[username]
(for user-own account-info for now,
 possibly changed to user-page (like FB) in the future)

File: src/components/Header.js [hunk 1]
```diff
@@ -17,11 +18,17 @@ export default (props) => {
...
 
-  let goUserHome = (userID) => {
+  let goUserHome = () => {
+    window.location.href = '/user/' + userID
+  }
+
+  let renderUserHome = () => {
     if(!userID || userID === PTT_GUEST) {
-      return
+      return <Empty />
```
Comment (chhsiao1981) [inline src/components/Header.js:27]: We would like to change to `<a className={'pull-right' + styles['navbar-link']} href="/login">hi~guest</a>`

**Review by chhsiao1981 — COMMENTED** [review]
We would like to have "/login" on the top-right if user is PTT_GUEST.
(and have "/login" as HomePage (will be renamed as LoginPage in the future) in Dev.js)

File: src/components/Header.js (hunk 1 above)
Comment (solring) [reply src/components/Header.js:27]: Got it~~

File: src/routes/All.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ export default (props) => {
     <Router>
       <Switch>
         <Route exact path="/" component={HomePage} />
+        <Route exact path="/login" component={HomePage} />
```
Comment (solring) [inline src/routes/All.js:12]: Since now home page is the same with login page, it's now simply directed to the same component.

Comment (chhsiao1981) [reply src/routes/All.js:12]: /lgtm

**Pull request merged**